src/reducers/__tests__/deleteEmptyThought.ts

```diff
@@ -248,28 +251,32 @@ describe('mount', () => {
   beforeEach(createTestApp)
   afterEach(cleanupTestApp)
 
-  it('after deleteEmptyThought, caret should move to end of previous thought', () => {
-    store.dispatch([
-      { type: 'newThought', value: 'apple' },
-      { type: 'newThought' },
-      { type: 'deleteEmptyThought' }
-    ])
-    jest.runOnlyPendingTimers()
+  it('after deleteEmptyThought, caret should move to end of previous thought', async () => {
+    await fakeTimer.runAsyncCode(() => {
```

Comment:
Could you explain the reason for adding `runAsyncCode` here? I'd like to understand this more.